packages/manager/core/generator/app/conditional-templates/onboarding/onboarding.test.ts.hbs

```diff
@@ -0,0 +1,16 @@
+import { expect, test } from '@playwright/test';
+import { login, logout } from '../../../../../playwright/login';
+import * as config from '../../../../../playwright/config.json';
+
+test('Onboarding test', async ({ page }) => {
+  await page.goto(config.appUrl);
+  await login(page);
+  const onboardingPath = '/#/onboarding';
+  const absoluteUrl = new URL(onboardingPath, config.appUrl).href;
+  await page.click(':nth-child(1) > a');
+  await page.goto(absoluteUrl, { waitUntil: 'networkidle' });
+  await page.waitForSelector('h1');
+  const h1Text = await page.textContent('h1');
+  expect(h1Text).toContain('Onboarding page');
```

Comment:
missing await here and same comment as in[packages/manager/core/generator/app/conditional-templates/default/home.test.ts.hbs](https://github.com/ovh/manager/pull/9365/files#diff-cdce3280f40d1da5bb6a2a5198f6dec8129b9c4b83a9f06c27dc4a9bd4049519)